structure_tests/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+export VERSION=$1
+
+if [ -z "$1" ]; then
+  echo "Please provide valid JSON schema version to tag image."
```

Comment:
This should just read "version" to tag the image, right? One image supports multiple versions I think.